ui/main/src/app/modules/cards/components/card-details/card-details.component.html

```diff
@@ -0,0 +1,26 @@
+<!-- Copyright (c) 2021, RTE (http://www.rte-france.com)              -->
+<!-- See AUTHORS.txt                                                       -->
+<!-- This Source Code Form is subject to the terms of the Mozilla Public   -->
+<!-- License, v. 2.0. If a copy of the MPL was not distributed with this   -->
+<!-- file, You can obtain one at http://mozilla.org/MPL/2.0/.              -->
+<!-- SPDX-License-Identifier: MPL-2.0                                      -->
+<!-- This file is part of the OperatorFabric project.                      -->
+
+<div *ngIf="card && cardState" style="font-size:13px;" [hidden] = "cardLoadingInProgress" >
+  <of-detail   [cardState]="cardState" [card]="card" [childCards]="childCards"
+             [currentPath]="_currentPath" [parentModalRef]="parentModalRef" [screenSize]="screenSize" [displayContext]="displayContext">
+  </of-detail>
+</div>
+
+<div *ngIf="cardLoadingInProgress" [class.opfab-card-loading]="!isSmallscreen()"  [class.opfab-card-loading-small-screen]="isSmallscreen()">
```

Comment:
Why did you choose to use use the `hidden` attribute for the first div and an ngIf for the second div?
I'm asking as I think ngIf and hidden are not the same thing for cypress for example.

Reply:
generally i use *ngIf by default and hidden when *ngIf does not match my needs 

On the first div , it is needed to keep the component instantiate between updates so i use hidden . Otherwise when we are watching a card in fullscreen mode and the card is updated , il goes back in smallscreen mode as the component is destroy and recreated . 

For the second div , it's my default choice 

*ngIf by default seems the choice recommended by angular : 
https://stackoverflow.com/questions/51317666/when-should-i-use-ngif-over-hidden-property-and-vice-versa/51317774